### PR TITLE
Enforced lint styling of using single quotes.

### PR DESCRIPTION
**Pull request by bhamodi**
@jordangarcia

### CODE REVIEW

File: tests/evaluator-tests.js [hunk 1]
```diff
@@ -21,44 +21,44 @@ describe('Evaluator', () => {
...
     })
 
-    it("return entire map when passed `[]`", () => {
+    it('return entire map when passed `[]`', () => {
       var result = evaluator.evaluate(state, [])
       expect(Immutable.is(state, result)).toBe(true)
     })
 
-    it("['top']", () => {
+    it('[\'top\']', () => {
```
Comment (jordangarcia) [inline tests/evaluator-tests.js]: this is why i like double quotes =P

**Comment by jordangarcia** [comment]
I am fine with removing the single vs double quotes.

Or encouraging people to use single quotes in source files.

File: tests/evaluator-tests.js (hunk 1 above)
Comment (bhamodi) [inline tests/evaluator-tests.js]: Haha, I know I know. Escaped characters are not too much of a hassle. :)

Comment (dtothefp) [inline tests/evaluator-tests.js]: @jordangarcia I like enforcing consistency over escaping. I think with eslint you can do `[2, "single", "avoid-escape"]` which would result in:

var double = "double" //exception
var single = 'single' //all good
var noEscape = '<div class="no-escaping">' //all good

Comment (jordangarcia) [inline tests/evaluator-tests.js]: ooh i like that,

what about

```js
var a = "foo's";
```

Comment (dtothefp) [inline tests/evaluator-tests.js]: I don't know how esLint works but jshint will just work with consistency in your project. I think you have to choose one over the other so if you wanted double over single you would do 

```js
["double", "avoid-escape"]
```

but I prefer single

Comment (bhamodi) [inline tests/evaluator-tests.js]: Ya same. @jordangarcia, would you be cool with this?
```js
it('["top"]', () => {
```

**Comment by jordangarcia** [comment]
LGTM

**Pull request merged**